app/controllers/registrants_controller.rb

```diff
@@ -1,34 +1,20 @@
 class RegistrantsController < ApplicationController
-  def index
-    @registrants = Registrant.all
-    render 'index.html.erb'
-  end
+  skip_before_action :authenticate_user!
 
   def new
     @registrant = Registrant.new
   end
 
   def create
-    @registrant = Registrant.new(first_name: params[:first_name], last_name: params[:last_name], affiliation: params[:affiliation], occupation: params[:occupation],  email: params[:email])
-    @registrant.save 
-  end
-
-  def show
-    @registrant = Registrant.find_by(id: params[:id])
-    render 'show.html.erb'    
+    registrant = Registrant.new(first_name: params[:first_name], last_name: params[:last_name], affiliation: params[:affiliation], occupation: params[:occupation],  email: params[:email], phone: params[:phone])
```

Comment:
Look at `params.require` from other controllers

Goal is to `Registrant.new(registrant_params)`